src/loaders/compute_tools/checkm2/checkm2.py

```diff
@@ -18,7 +29,8 @@ def _run_checkm2(ids_to_files: Dict[str, Path], output_dir: Path, threads: int,
                '--output-directory', str(output_dir),
                '--threads', str(threads),
                '--force',  # will overwrite output directory contents
-               '--input'] + [str(v) for v in ids_to_files.values()]
+               '--input'] + [str(getattr(v, loader_common_names.META_SOURCE_FILE))
```

Comment:
You might as well just use the `.` syntax here, unless there's something I'm not understanding / remembering

Reply:
Same for more spots below

Reply:
Please note v is a NameTuple here. 

.syntax will result in AttributeError and we have discussed about this in the past PR.

Reply:
That was a specific case IIRC. Declaring the tuple names with constants from loader_common_names probably isn't necessary for this case. You can just do something like

```
GenomeTuple = namedtuple("GenomeTuple", ["file", "data_id"])
```

...and then access like `genome_tuple.file`

Reply:
I remember doing it with getattr for the fatal tuple but I don't remember why we did it that way, other than the original syntax of `fatal_tuple.loader_common_names.ERROR` or whatever wouldn't work

Reply:
Updated!

Reply:
:+1: